docs/core/extensions/sslstream-best-practices.md

```diff
@@ -0,0 +1,107 @@
+---
+title: TLS/SSL best practices
+description: Learn best practices to using SslStream in .NET
+author: rzikm
+ms.author: rzikm
+ms.date: 10/25/2022
+---
+
+# TLS/SSL best practices
+
+TLS (Transport Layer Security) is a cryptographic protocol designed to secure communication between two computers over the internet. The TLS protocol is exposed in .NET via the <xref:System.Net.Security.SslStream> class.
+
+This article presents best practices for setting up secure communication between client and server and assumes .NET Core version 3.1 or later. For best practices for .NET Framework, see [Transport Layer Security (TLS) best practices with the .NET Framework](/dotnet/framework/network-programming/tls).
+
+## Selecting TLS version
+
+While it is possible to specify the version of the TLS protocol to be used via the SslProtocols property, it is recommended to defer to the operating system settings by using <xref:System.Net.Security.SslProtocols.None> value (this is the default).
+
+Deferring the decision to the OS automatically uses the most recent version of TLS available and lets the application pick up changes after OS upgrades. The operating system may also prevent use of TLS versions which are no longer considered secure.
```

Comment:
While this true, modifying OS is not always possible or practical. I'm wondering if we can craft some example to enforce minimal version. That can perhaps be done post handshake...? 

cc: @baronjs for any suggestions.

Reply:
Our API doesn't support "minimal version", only "these versions", which makes it brittle when a new version comes online (e.g. specifying `Tls12 | Tls13` today means it ends up boxing out Tls14 in the future).

I think I agree with this saying simply "specify None, and let the OS decide"... which should call just enough attention to the option that someone could figure out they could specify specific versions.